github/ci/prow/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits.yaml

```diff
@@ -41,7 +41,7 @@ presubmits:
       optional: true
       decorate: true
       decoration_config:
-        timeout: 3h
+        timeout: 4h
```

Comment:
why do we need it on pull-e2e-cluster-network-addons-operator-lifecycle-okd? I thought this lane is not running